### PR TITLE
Marciflip gtest

### PR DESCRIPTION
**Pull request by tgiroux**
## Description

GTest for marciflip. I think having a single test case is appropriate here.
Couldn't reuse cubes from marcical tests due to their shape and summing mode.
Also moved marcical test cubes to their own directory.

## Motivation and Context

Test translations.

## How Has This Been Tested?

New test passing on osx.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/tests/FunctionalTestsMarciflip.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  EXPECT_EQ( (int)dims["Bands"], 3 );
+
+  PvlGroup &inst = label->findGroup("Instrument", Pvl::Traverse);
+  EXPECT_EQ( (int)inst.findKeyword("DataFlipped"), 0 );
+
+  std::unique_ptr<Histogram> outHist (outCube.histogram());
+  EXPECT_NEAR( outHist->Average(), 50.552, 1e-3 );
+  EXPECT_NEAR( outHist->Sum(), 4853, 1 );
+  EXPECT_EQ( outHist->ValidPixels(), 96 );
+  EXPECT_NEAR( outHist->StandardDeviation(), 11.63829, 1e-5 );
```
Comment (jessemapel) [inline isis/tests/FunctionalTestsMarciflip.cpp]: This doesn't actually check that the framelets were flipped. The statistics should be the exact same for both cubes as this application just moves pixels around.

You should check that the first 8 lines in the input equal the last 8 lines in the output.

**Review by jessemapel — COMMENTED** [review]
Looks good, with one change. We should probably have two tests, with and without summing, but the original didn't so it's okay for the conversion not to.

File: isis/tests/FunctionalTestsMarciflip.cpp (hunk 1 above)
Comment (tgiroux) [reply isis/tests/FunctionalTestsMarciflip.cpp]: Was having trouble doing that comparison with Bricks. Took the time this morning to implement with Portals and it works just fine.

**Review by jessemapel — APPROVED** [review]
Looks great!

**Comment by jessemapel** [comment]
This can be merged once CI tests pass

**Pull request merged**